dist/index.js

```diff
@@ -193,6 +197,15 @@ async function runScan({
 
   cmdArgs.push("-o", outputFormat);
 
+  // always output to a file, this is read later to print table output
+  if (!outputFile) {
+    outputFile = path.join(
+      fs.mkdtempSync(path.join(os.tmpdir(), "grype-")),
+      "output",
+    );
+  }
+  cmdArgs.push("--file", outputFile);
```

Comment:
How does this interact with `output`? I had the impression we were moving towards `output` and way from `file`.

Reply:
We support `--output <format> --file <output-file>` in both Syft and Grype. This action doesn't allow multiple outputs, so it should work fine. I don't think there is a concrete plan to remove the `--file` option, since it's a part of Syft 1.0 already. If you feel strongly this should be using the `--output` flag, I can update it.

Reply:
I take this back, allowing for multiple outputs would be nice, so a user could get both a table printed to the logs and a SARIF report uploaded. I'll update this a bit.

Reply:
@kzantow did you have additional updates to make here? I think you were going to update to use `--output` or something?